proton/pins.json

```diff
@@ -0,0 +1,22 @@
+{
+    "domain_list": [
```

Comment:
In fact this should rather be a map:
```json
{
    "domain_hashes": {
        "api.protonvpn.ch": [
             "...",
             "...."
        ]
    }
}

Reply:
Yes, I will convert this to a constant instead.